numpy/core/code_generators/cversions.txt

```diff
@@ -67,5 +67,7 @@
 # Version 16 (NumPy 1.23)
 # NonNull attributes removed from numpy_api.py
 # Version 16 (NumPy 1.24) No change.
-# Version 16 (NumPy 1.25) No change.
 0x00000010 = 04a7bf1e65350926a0e528798da263c0
+
+# Version 17 (NumPy 1.25) No actual change.
+0x00000011 = ca1aebdad799358149567d9d93cbca09
```

Comment:
The min-version macros do goes into the hash right now.  So removing them will also change it, but that seems fine?

Reply:
Only for >= 1.25, right? Should be fine I'd think.